diesel/src/expression/count.rs

```diff
@@ -9,6 +9,27 @@ use types::BigInt;
 /// As with most bare functions, this is not exported by default. You can import
 /// it specifically as `diesel::expression::count`, or glob import
 /// `diesel::expression::dsl::*`
+///
+/// # Example
+///
+/// ```rust
+/// # #[macro_use] extern crate diesel;
+/// # include!("src/doctest_setup.rs");
+/// # use diesel::expression::dsl::*;
+/// #
+/// # table! {
+/// #     users {
+/// #         id -> Integer,
+/// #         name -> VarChar,
+/// #     }
+/// # }
```

Comment:
No, it is required. Without it:

```
error[E0433]: failed to resolve. Use of undeclared type or module `users`
  --> src/doctest_setup.rs:141:6
   |
141|     (users)
   |      ^ Use of undeclared type or module `users`
```

I thought about moving the table!  macro into the `doctest_setup.rs` module, but in some places the invocation is being displayed in the docs, so I didn't touch it and instead copied it around.